src/glint_ffi.mjs

```diff
@@ -0,0 +1,4 @@
+export function exit(code) {
+  import("node:process").then((process) => process.exit(code));
+  return undefined;
+}
```

Comment:
Without catching the error, the `Promise` will throw, and it will trigger the global unhandled Promise handler.
```suggestion
export function exit(code) {
  import("node:process")
    .then((process) => process.exit(code))
    .catch((error) => {});
  return undefined;
}
```

Reply:
oh nice! shows how little i know about JS